cs-config/cs_config/functions.py

```diff
@@ -204,9 +215,20 @@ def comp_output(base_ss, base_params, reform_ss, reform_params,
               "media_type": "PNG",
               "title": 'Labor Supply in Baseline and Reform Policy',
```

Comment:
@jdebacker . Shouldn't this "title" be `'Consumption in Baseline and Reform Policy'`? And why not have two figures, one for consumption and one for labor supply? It seems like this PR is removing the labor supply picture and substituting it with consumption.